{{ cookiecutter.formal_name }}/{{ cookiecutter.formal_name }}/main.m

```diff
@@ -73,26 +65,6 @@ int main(int argc, char *argv[]) {
         PySys_SetArgv(argc, python_argv);
 
         @try {
-            if (nslog_script == NULL) {
-                NSLog(@"No Python NSLog handler found. stdout/stderr will not be captured.");
-                NSLog(@"To capture stdout/stderr, add 'std-nslog' to your app dependencies.");
-            } else {
-                NSLog(@"Installing Python NSLog handler...");
-                FILE *fd = fopen(nslog_script, "r");
-                if (fd == NULL) {
-                    NSLog(@"Unable to open nslog.py; abort.");
-                    crash_dialog(@"Unable to open nslog.py");
-                    exit(-1);
-                }
-
-                ret = PyRun_SimpleFileEx(fd, nslog_script, 1);
-                fclose(fd);
-                if (ret != 0) {
-                    NSLog(@"Unable to install Python NSLog handler; abort.");
-                    crash_dialog(@"Unable to install Python NSLog handler.");
-                    exit(ret);
-                }
-            }
```

Comment:
As I commented on #7 - we can't completely dump the NSLog script, as it's not *just* about the exception handling. This PR does mean we can remove the custom exception hook though.

Reply:
Yeah, that was a copy-paste gone wrong...